### ISSUE TITLE
Support  other RelationalDB 

### ISSUE THREAD

**Issue by rongfengliang** [SigNoz/signoz#941]
## Is your feature request related to a problem?

currently from the source code I see `signoz `  only support  SQLite database . maybe we can support more other database 

## Describe the solution you'd like

we can use  `jmoiron/sqlx` package   and some configuration  for support more database 

#### *Thank you* for your feature request – we love each and every one!

**Comment by pranay01** [comment]
@rongfengliang Curious to know. whats the use case for supporting more databases? As of now SQLite is primarily used to store internal metadata like dashboards layout, alert config, etc

**Comment by rongfengliang** [comment]
@pranay01 Thanks. with sqlite for dashboard and alert . if we have multi instance of query service maybe not good for share message

**Comment by ankitnayan** [comment]
@rongfengliang that is right. We have added the apis in an interface so that other databases like Postgres can implement them. We shall need non-embedded db when running multiple query-service instances querying the same relational DB as you mentioned already. 

We shall pick this up if some users face this issue. Currently, I don't see a need to run multiple query-services as query-service is not under heavy load usually.

**Comment by mindhash** [comment]
@dwelch2344 I have started working on this. You should see it soon in upcoming releases.

**Comment by ngudbhav** [comment]
Is this still under development?

**Comment by grandwizard28** [comment]
Yes @ngudbhav,
This is under active development. We have created a master issue to effectively track this: https://github.com/SigNoz/signoz/issues/7316

**Comment by therealpandey** [comment]
Postgres is now live. Here are the docs to configure it: https://signoz.io/docs/manage/administrator-guide/configuration/relational-database/.

Here is the announcement blog: https://signoz.io/blog/oss-improvements/.
